gslab_scons/_release_tools.py

```diff
@@ -234,11 +238,9 @@ def up_to_date(mode = 'scons', directory = '.'):
             raise ReleaseError('up_to_date(mode = git) must be run on a '
                                'git repository.')
 
-        # If mode = git, look for a line stating that sconsign.dblite has changed
+        # If mode = git, look for a line stating that nothing has changed
         # since the latest commit.
-        result = [out for out in output if re.search('sconsign\.dblite', out)]
-        result = [True for out in result if re.search('modified', out)]
-        result = not bool(result)
+        result = bool('nothing to commit, working tree clean' in output)
```

Comment:
good catch

Reply:
Thanks!

Reply:
So I'm getting this when I run `release` from a non-SCons directory.

```
gslab_scons._exception_classes.ReleaseError: SCons targets not up to date.
```

It should return this, no?

```
            raise ReleaseError('up_to_date(mode = scons) must be run on a '
                               'SCons directory.')
```

Reply:
@arosenbe just this point, then can you make a few test releases with the various modes and pull in? Thanks